Strata One/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "../profile.schema.json",
+  "name": "01node",
+  "identity": "6f00a741922b2935f8f61518092304e1c64847b5"

```

Comment:
This doesn't look like a Keybase identity string, but I won't hold up the merge for it.